changelogs/fragments/70319-reduce-ignored-module-sanity-tests.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fixes ``mode`` doc on ``apt_repository`` to remove ineffective default (https://github.com/ansible/ansible/pull/70319)
```

```suggestion
  - apt_repository - fixes ``mode`` doc to remove ineffective default (https://github.com/ansible/ansible/pull/70319).
```
I'm not sure whether there should be a changelog fragment for this change at all, though, since this doesn't change detectable behavior for users. That's just my personal opinion though :)